hack/tests/origin-conformance.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash -ex
+
+REAL="`realpath $0`"
+HERE="`dirname ${REAL}`"
```

Comment:
all our tests currently hard code `hack/tests` instead of $HERE, keep consistency for now